lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -2541,7 +2541,10 @@ private function newInstance($class)
     public function createEntity($className, array $data, &$hints = [])
     {
         $class = $this->em->getClassMetadata($className);
-        //$isReadOnly = isset($hints[Query::HINT_READ_ONLY]);
+
+        if ( ! $class instanceof ClassMetadata) {
+          return $entity;
```

Comment:
We use 4 spaces indentation. Also, `ClassMetadata` is always an instance of `Doctrine\ORM\Mapping\ClassMetadata`: the check should be removed (and yes, I know the metadata factory interface returns a `Doctrine\Common\Persistence\Mapping\ClassMetadata` as per contract)

Reply:
I've just added that because of Scrutinizer complaining about it